src/util/expander.js

```diff
@@ -31,16 +31,17 @@ export function expand(el, {
 	// set the property to the 'from' value
 	el.style[property] = from;
 
+	el.addEventListener('transitionend', function listener() {
+		unsetStyles(el, { property });
+		el.removeEventListener('transitionend', listener, true);
+		// finally, call the done callback after the transition
+		done();
+	}, true);
+
 	// ...and set the property to the measured value on the next tick so it animates w/ css
 	window.setTimeout(() => {
 		el.style[property] = propValue;
-	}, 1);
```

Comment:
The delay for this timeout still needs to be 1 so that this value is set on the next rendering loop instead of the current rendering loop (ditto for the same timeout in the collapse function)

Reply:
@emuvente a setTimeout of 0 should run in the next tick which works here (otherwise we wouldn't see the transition at all). Another way to init the transition would be to force a reflow calculation by asking for the offsetWidth or something like it before setting the property again. A more detailed explanation here https://stackoverflow.com/a/60686560